organizers/2024-03-13.html

```diff
@@ -0,0 +1,148 @@
+---
+layout: organizers
+---
+
+<!-- Syntax reference: https://github.com/hakimel/reveal.js/blob/master/index.html -->
+
+<section>
+    <h1>The London Django Meetup Group</h1>
+    <h2>March 13th, 2024</h2>
+    <img data-src="/img/django-logo-white-transparent.svg"
+         style="width: 40%; background: none;"
+         alt="Django logo"
+         class="plain">
+    <h2>Talks starting at 19:00!</h2>
+</section>
+
+<section>
+    <h1>👋 Welcome All!</h1>
+</section>
+
+<section>
+    <h1>Code of conduct</h1>
+    <h2>😃 Be Awesome to Each Other</h2>
+    <p>
+        The mission of The London Django Meetup Group is to provide a fun,
+        welcoming and professional environment for a diverse group of people.
+    </p>
+    <p>
+        Code of conduct:
+        <a href="https://www.djangolondon.com/code-of-conduct/">djangolondon.com/code-of-conduct</a>
+    </p>
+</section>
+
+<section>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+        <li><strong>Kraken Tech (part of the Octopus Energy Group)</strong></li>
+        <li><strong>Jetbrains</strong></li>
+    </ul>
+</section>
+
+<section>
+    <img data-src="/img/2022_KrakenV2_stacked_White.png"
+         style="width: 60%; border-radius: 10px"
+         alt="Kraken Tech"
+         class="plain">
+    <p>
+        <em>
+            Kraken is a revolutionary energy tech platform built with Django. Serving 25 million households worldwide.
+        </em>
+    </p>
+</section>
+
+<section>
+    <h1>📋 Today's Schedule</h1>
+    <h2>🐍 News</h2>
+    <h2>👩🏻‍💻 Talks</h2>
+    <h2>🎁 Prize Draw</h2>
+    <h2>💬 Open Discussion</h2>
+</section>
+
+<section>
+    <h1>Django News</h1>
+    <p>Weekly newsletter of blog posts, developments, talks, packages, etc.</p>
+    <p>django-news.com</p>
+</section>
+
+<section>
+    <h2>🔒 Django security releases</h2>
+    <p>5.0.3 and related backports fix a DoS vulnerability.</p>
+    <p>Upgrade monthly!</p>
+</section>
+
+<section>
+    <h2>🇪🇸 Djangocon Europe</h2>
+    <p>June 5th to 9th</p>
+    <p>Vigo, Spain</p>
+    <p>🐣 Earlybird tickets still on sale.</p>
+</section>
+
+<section>
+    <h2>🇺🇸 Djangocon US CfP</h2>
+    <p>Deadline: April 24</p>
+    <p>Submit your talk or workshop proposal now!</p>
+</section>
+
+<section>
+    <h2>🇨🇿  EuroPython 2024</h2>
+    <p>July 8-14, 2024</p>
+    <p>Prague, Czech Republic</p>
+    <p>Volunteer opportunities open, CfP until <strong>tomorrow morning</strong>.</p>
+</section>
+
+<section>
+    <h2>🇮🇹 PyCon Italia 2024 - Seeking Sponsors!</h2>
+    <p>Dates: May 22-25, 2024</p>
+    <p>Florence, Italy</p>
+    <p>Join over 800 attendees from 45 countries for 100+ talks and workshops.</p>
+</section>
+
+<section>
+    <h1>🎤 Talks</h1>
+    <strong>WASM-powered Django apps with PyScript</strong><br>
+       <em>Valerio Maggio</em>
```

Comment:
```suggestion
       <em>Valerio Maggio</em><br><br>
```
Add a br otherwise they are on the same line, it's a bit odd :D

Reply:
solved with `<p>` instead